app/controllers/api/v4/receipts_controller.rb

```diff
@@ -17,6 +17,18 @@ def create
         render "show"
       end
 
+      def destroy
+        @receipt = Receipt.find(params[:id])
+
+        if policy(@receipt).destroy?
+          @receipt.destroy!
+          render json: { message: "Receipt successfully deleted" }, status: :ok
+        else
+          render json: { error: "You are not authorized to delete this receipt" }, status: :forbidden
+        end
+      end
+
+
```

Comment:
Any reason we aren't directly doing `authorize @receipt`

Reply:
Nope, you're right it is more consistent to use authorize instead of using policy.